sympy/stats/crv_types.py

```diff
@@ -1227,6 +1229,71 @@ def GammaInverse(name, a, b):
     return rv(name, GammaInverseDistribution, (a, b))
 
 #-------------------------------------------------------------------------------
+# Gompertz distribution --------------------------------------------------------
+
+class GompertzDistribution(SingleContinuousDistribution):
+    _argnames = ('eta', 'b')
+
+    set = Interval(0, oo)
+
+    @staticmethod
+    def check(eta, b):
+        _value_check(eta > 0, "eta must be positive")
+        _value_check(b > 0, "b must be positive")
+
+    def pdf(self, x):
+        eta, b = self.eta, self.b
+        return b * eta * exp(b * x) * exp(eta) * exp(-eta * exp(b * x))
```

Comment:
This is probably in line with PEP 8, but I think it would be more readable if the spaces in the arguments of ``exp`` were omitted.